src/types/index.ts

```diff
@@ -664,36 +664,36 @@ export type TransactionMultiEd25519Signature = {
 
 export type TransactionMultiAgentSignature = {
   type: "multi_agent_signature";
-  sender: AccountSignature;
+  sender: TransactionAccountSignature;
```

Comment:
should be marked as [`Breaking`] in changelog

Reply:
I'm positive nobody is using this type 😆 
but can add it for good measure

Reply:
sdk users do... wild things